about.md

```diff
@@ -1,6 +1,14 @@
 ---
 layout: page
-title: About me 
+title: My name is Joffrey and I <3 development 
 ---
 
-This is a static page. It could be an 'about page' if you'd like.
+My name is Joffrey Kern, I'm living in Strasbourg, France and working for Euro Information Developement on a project called LyfPay as .NET Architect. I'm also co-founder of the [Microsoft User Group Strasbourg](https://www.meetup.com/MugStrasbourg/). I'm a [developer](./projects.html) before anything else and I like to share my knowledge with this [blog](./blog) or [conferences](./speaker.html).
```

Comment:
Je voulais aussi parler de "I'm living" trop tard ^^.

Sinon est-ce que tu ne veux pas plutôt dire "I would like to share" ? Et je mettrais plutôt "through this blog" aussi.